<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WBalcone</class>
 <widget class="QWidget" name="WBalcone">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1154</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="cameraBtn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>101</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>Camera</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cassapancaBtn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>380</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Cassapanca</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openSpaceBtn">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>190</y>
     <width>91</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string>OpenSpace</string>
   </property>
  </widget>
  <widget class="QPushButton" name="floorLeftBtn">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>210</y>
     <width>81</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string>FloorLeft</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scafRight1Btn">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>210</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>ScafRight1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scafLeft1Btn">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>210</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>ScafLeft1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scafLeft2Btn">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>280</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>ScafLeft2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scafLeft3Btn">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>350</y>
     <width>80</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>ScafLeft3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scafLeft4Btn">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>420</y>
     <width>80</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>ScafLeft4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scafRight2Btn">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>280</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>ScafRight2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scafRight3Btn">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>350</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>ScafRight3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scafRight4Btn">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>420</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>ScafRight4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="allObjsBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>All Objects</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
